dair_pll/urdf_utils.py

```diff
@@ -335,15 +361,19 @@ def represent_multibody_terms_as_urdfs(multibody_terms: MultibodyTerms,
                     continue
                 body_index = all_body_ids.index(body_id)
                 body_geometry_indices = \
-                    multibody_terms.geometry_body_assignment[
-                        body_id]
+                    multibody_terms.geometry_body_assignment[body_id]
                 body_geometries = [
                     cast(CollisionGeometry,
                          multibody_terms.contact_terms.geometries[index])
                     for index in body_geometry_indices
                 ]
+                body_frictions = \
+                    multibody_terms.contact_terms.friction_coefficients[
+                        body_geometry_indices
+                    ]
                 fill_link_with_parameterization(element, pi_cm[body_index, :],
-                                                body_geometries, output_dir)
+                                                body_geometries, body_frictions,
+                                                output_dir)
 
         register_namespace('drake', 'https://drake.mit.edu/')
```

Comment:
Now that the namespace is included in the static variables of `urdf_utils.py`, these hardcoded strings should probably be moved to the top of the file